src/commands/deploy/latest.js

```diff
@@ -459,6 +459,12 @@ export default async function main(
       error(message);
     }
 
+    if (err.code === 'size_limit_exceeded') {
+      const message = `File size limit exceeded (${bytes(err.sizeLimit)})`;
```

Comment:
is this input to be trusted?

Reply:
@juliangruber Let's add a check for whether the property is there.